util/version/version.go

```diff
@@ -33,6 +33,14 @@ var (
 	KubeSemverTolerant = regexp.MustCompile(`^v?(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)([-0-9a-zA-Z_\.+]*)?$`)
 )
 
+func mustParseTolerant(s string) semver.Version {
```

Comment:
given this function seems used only in test, can we move it to version_test

Reply:
Sure, I will add it to version_test